base/strings/basic.jl

```diff
@@ -414,10 +455,7 @@ julia> chr2ind(str, 2)
 3
 ```
 """
-function ind2chr(s::AbstractString, i::Integer)
-    s[i] # throws error if invalid
-    unsafe_ind2chr(s, i)
-end
+ind2chr(s::AbstractString, i::Integer) = length(s, 1, i)
```

Comment:
Should we rename/remove `ind2chr` and `chr2ind`? There are discussions about renaming `sub2ind` and `ind2sub`, so the "2" pun could be completely removed from the API. Deprecations are trivial with the new methods.

Reply:
Yes, that was my thought as well but I figured I'd leave it for another PR since this one is already so large.